### PR TITLE
removes messagelist instead of only list elements in modal window

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-17 at 10 15 09|screen shot 2015-11-17 at 12 00 44|

### CODE REVIEW

**Comment by FinalAngel** [comment]
No way we can control this via css from adminstyle?

**Comment by lory87** [comment]
@FinalAngel no because empty selector is not working because of white spaces

**Comment by FinalAngel** [comment]
alright LGTM

File: cms/static/cms/js/modules/cms.modal.js [hunk 1]
```diff
@@ -880,13 +881,14 @@ var CMS = window.CMS || {};
                     CMS.API.Toolbar.hideLoader();
 
                     // show messages in toolbar if provided
+                    messageList = $(iframe.contents().find('.messagelist');
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.modal.js]: i'm pretty sure this throws an error

File: cms/static/cms/js/modules/cms.modal.js [hunk 2]
```diff
@@ -880,13 +881,14 @@ var CMS = window.CMS || {};
...
                     messages = iframe.contents().find('.messagelist li');
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.modal.js]: here can reuse the `messageList`

File: cms/static/cms/js/modules/cms.modal.js (hunk 1 above)
Comment (lory87) [inline cms/static/cms/js/modules/cms.modal.js]: yes corrected it at this moment. Saw it as well.

File: cms/static/cms/js/modules/cms.modal.js [hunk 3]
```diff
@@ -880,13 +881,14 @@ var CMS = window.CMS || {};
                     CMS.API.Toolbar.hideLoader();
 
                     // show messages in toolbar if provided
+                    messageList = iframe.contents().find('.messagelist');
                     messages = iframe.contents().find('.messagelist li');
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.modal.js]: here can reuse the `messageList`

File: cms/static/cms/js/modules/cms.modal.js [hunk 4]
```diff
@@ -880,13 +881,14 @@ var CMS = window.CMS || {};
                     CMS.API.Toolbar.hideLoader();
 
                     // show messages in toolbar if provided
-                    messages = iframe.contents().find('.messagelist li');
+                    messageList = iframe.contents().find('.messagelist');
+                    messages = messageList.find ('li');
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.modal.js]: no need for a space here

**Comment by vxsx** [comment]
:+1:

**Pull request merged**